style.css

```diff
@@ -436,9 +431,195 @@ textarea {
     margin: 0 1.8rem;
   }
 
-  .box2 h1 {
+  .Greetings h1 {
     font-size: 40px;
     max-width: 300px;
     line-height: 3rem;
   }
 }
+
+@media screen and (min-width: 768px) {
+  .box2 {
+    background-image: url(imgs/Header\ bg.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    margin-bottom: 100px;
+  }
+
+  .about {
+    display: grid;
+    grid-template-columns: 300px 2fr;
+    padding-top: 3rem;
+    margin-bottom: 0;
+  }
+
+  .form-group {
+    background-image: url(imgs/contact\ form\ background\ shapes@2x.svg);
+    background-repeat: no-repeat;
+    background-position: 100% 100px;
+    background-size: cover;
```

Comment:
- [ ] I would suggest you to consider reducing the width of the form container so as to match the width on the guiding template. The width of your form container is too wide and that has made your design not to be inline with the design speculation. Kindly look into the figma design for that form part and work your design alongside that design.
- [ ] Also, don't forget to position the `Get in Touch` button to the center on the desktop version of the project.

### Your Design
![your-contact-form](https://user-images.githubusercontent.com/26597432/206661335-4145490d-d36b-48aa-a8f5-1ba5f672e3c9.png)

### Figma Design
![figma-contact-form](https://user-images.githubusercontent.com/26597432/206661368-a6ce6de7-e9b5-426f-8919-5e22770e3cd6.png)